src/api.raml

```diff
@@ -497,6 +497,13 @@ traits:
         is:
           - paged: { resources: Collections }
           - subjected: { resources: Collections }
+        queryParameters:
+            containing:
+                description: |
+                    Only return collections containing at least one of these items.
+                type: string[]
+                required: false
+                example: [article/1234, interview/5678]
```

Comment:
I like the idea of consistently referring to a content type + its ID. So this would mean small changes to Recommendations and Metrics (to support both, and deprecate using slashes). /cc @lsh-0